bin/autorun.sh

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env bash
+
+## this is the default autorun script
+## file should run automatically after init.sh completes.  
+## this script downloads your configured run.env, system.env and any custom container requests
+
+INSTALL_DIR_TEMP="$( cd "$( dirname "${BASH_SOURCE[0]}" )/.." >/dev/null 2>&1 && pwd )"
+
+## retrieve the bucket name you sent the instance earlier
+BUCKET=$(cat $INSTALL_DIR_TEMP/bucket.txt)
+
+## load some variables
+source $INSTALL_DIR_TEMP/bin/scripts_wrapper.sh
+source $INSTALL_DIR_TEMP/bin/activate.sh
+
+
+## get the updatated run.env and system.env files you created and stashed in s3
+aws s3 cp s3://$BUCKET/run.env $INSTALL_DIR_TEMP/run.env
+aws s3 cp s3://$BUCKET/system.env $INSTALL_DIR_TEMP/system.env
```

Comment:
See comment above; let's have a list of s3-urls, and then let's iterate over them and use aws s3 sync.

Reply:
i modified based on your suggestion to have a prefix in s3 that is dedicated to files that need copied over (node-config), and then use aws s3 sync.